src/test/repaymentsTest.js

```diff
@@ -2,15 +2,48 @@
 import chai, { should } from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../app';
+import generateUserToken from '../utils/helpers/generateUserToken';
 
 should();
 chai.use(chaiHttp);
 
 describe('Repayments', () => {
   describe('GET /loans/:loanId/repayments', () => {
+    it('it should fail if there is no token in the header', (done) => {
```

Comment:
Your test description does not necessarily need `it`. Your it blocks is doing that anyways
```suggestion
    it('should fail if there is no token in the header', (done) => {
```

Reply:
I've made the changes